pkg/controller/artifacts/crd-network.yaml

```diff
@@ -31,6 +31,17 @@ spec:
               type: string
             vpcID:
               type: string
+            service:
+              type: object
+              properties:
+                ipam:
+                  type: string
+                  enum: ["Arktos", "Kubernetes", "External"]
```

Comment:
Maybe combine "Arktos" and "Kubernetes" to a generic term "System"?   system v.s external

Reply:
what really matters is being "External" or not (kubernetes, arktos or system all treated equally). For now, it seems "Arktos" sounds most understandable. Anyway, if a more general value is more suitable in the future, change (or adding) won't affect implementation much.